experimenter/experimenter/nimbus_ui_new/views.py

```diff
@@ -46,6 +46,26 @@ def get_form_kwargs(self):
 
 
 class RenderResponseMixin:
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        show_errors = self.request.GET.get("show_errors", "") == "true"
+        errors = self.get_object().get_invalid_fields_errors()
+        form = self.get_form()
+
+        validation_errors = {}
+
+        if form and show_errors:
+            for field, error in errors:
+                if field in form.fields:
+                    if field not in validation_errors:
+                        validation_errors[field] = []
+                    validation_errors[field].append(error)
+
+        context["validation_errors"] = validation_errors
+
+        return context
```

Comment:
We should put this into its own mixin and only explicitly include it on the pages that need it.  If we include it in this mixin, then there's a bunch of pages that'll be doing the review serializer work that might not need it.  Also I see your point here about checking against the form, but since we're actually looking the individual fields up by name in the templates, we don't really need to do this filtering at all.  I'd just generate the invalid field errors, dump them right into the context, the template can pull out the fields it cares about, and that'll be it.